### ISSUE TITLE
Problem with saphira?

### ISSUE THREAD

**Issue by eehpcm** [openstreetmap/operations#264]
Over 24 hours ago I added a [house](https://www.openstreetmap.org/way/663861395#map=19/52.21387/-4.36185) which still isn't showing up at any zoom level where it should be visible.  I asked elsewhere and kocio said it was showing up.  It shows up on Andy Townsend's useful map.  It shows up for the query tool (obviously, from the link I just gave).  I conclude that it's made it into the db.

If I understand things right (probably not) and used the correct tool, the tile server debug says that, for me, saphira is the cache and yevaud is the renderer.

**Comment by tomhughes** [comment]
I'm sure it will appear in due course.

**Comment by tomhughes** [comment]
Replication had stalled on yevaud - it's catching up now but it will take a few hours.

**Comment by tomhughes** [comment]
There was no need to deliberately edit it - the replication has almost caught up now and it is dirtying anything that has changed as it goes!

That said the server is quite busy as a result so it may not rerender everything immediately.

**Comment by eehpcm** [comment]
@tomhughes 

> There was no need to deliberately edit it

Ah, sorry.  After I'd added that problem house I then added about 20 houses nearby.  None of them were showing up, so I reported the issue here.

Then you said yevuad was catching up.  I checked about three hours after you said that and those 20-ish houses had shown up but that problem house hadn't.  I assumed yevuad had caught up but that particular house was stuck for some weird reason.  My error.

A few hours ago I added a couple more, which haven't turned up.  So the server does indeed appear to be a little busy.
